RELEASE.md

```diff
@@ -12,10 +12,12 @@
 * CLI commands from sources with the same name will show under one list in the help screen.
 * The setup of a Kedro project, including adding src to path and configuring settings, is now handled via the `bootstrap_project` method.
 * Invoked `configure_project` if a `package_name` is supplied to `KedroSession.create`. This is added for backward-compatibility purpose to support workflow that creates a `Session` manually. It will only be removed in `0.18.0`.
+* `PartitionedDataSet` now supports lazily materializing data on save.
 
 ## Minor breaking changes to the API
 
 ## Thanks for supporting contributions
+[Lou Kratz](https://github.com/lou-k)
```

Comment:
Could you move this to the section for `0.17.4`?

Reply:
👍